app/views/comments/create.js.erb

```diff
@@ -1,4 +1,4 @@
-let added_comment = $("<%= j(render 'comments/comment', comment: @comment) %>");
```

Comment:
Very interesting  😯.  I guess every time we render we will declare the `added_comment" variable again, and this will raise `Identifier [variable_name] has already been declared` because of **let** block scoping behavior. I would keep this in mind as well.